app/assets/javascripts/proxy_status.js

```diff
@@ -1,5 +1,5 @@
 $(document).on('ContentLoad', function() {
-  $('.nav-tabs a').on('shown.bs.tab', refreshCharts);
+  // $('.nav-tabs a').on('shown.bs.tab', refreshCharts);
```

Comment:
might break other parts of the application. currently disabled because the code causes side effects with interfere with the this implementation. conflict must be resolved before merge.

The following PR s should take care of this: Fixes #14817 - load JS only in relevant pages #3466